test/python/scheduler/test_basic_scheduler.py

```diff
@@ -1145,29 +1086,33 @@ def test_inst_sched_map_get_measure_0(self):
         is actually Schedule for just qubit_0"""
         sched_from_backend = self.backend.instruction_schedule_map.get("measure", [0])
         expected_sched = Schedule(
-            (0, Acquire(1472, AcquireChannel(0), MemorySlot(0))),
```

Comment:
@mtreinish can you take a look at these tests? I don't have enough exposure to pulse scheduling to do a proper review